src/main/webapp/jsp/feedbackSubmissionEdit.jsp

```diff
@@ -134,7 +134,18 @@
                         %>
                         </select>
                     </div>
-                    <div <%=(question.isRecipientNameHidden()) ? "class=\"col-sm-12\"" : "class=\"col-sm-10\""%>>
+                    <div 
+                    <% if (question.isRecipientNameHidden()){
+                    	   out.println("class=\"col-sm-12\""); 
+                       }
+                       else if(numOfResponseBoxes == maxResponsesPossible){
```

Comment:
Oops sorry, I did not notice this earlier. But try to keep the 'else' on the same line as the curly bracket. You can look at our coding standard here: https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true#h.156j9wx7tamx